game-engine/domain/src/main/java/za/co/entelect/challenge/entities/TowerDefenseGameMap.java

```diff
@@ -108,11 +108,6 @@ public void deconstructBuilding(Building building) {
 
     public void removeBuilding(Building building) {
         buildings.remove(building);
-        try {
-            getPlayerOpponent(building.getPlayerType()).addScore(building.getDestroyMultiplier());
-        } catch (Exception e) {
-            log.error(e);
-        }
```

Comment:
On building removal the opponent gets the multiplier as score. This is completely wrong.

Building destruction score is not awarded, only building damage score until the building is at 0hp